hazelcast/src/main/java/com/hazelcast/jet/impl/connector/ReadJdbcP.java

```diff
@@ -104,7 +105,7 @@ public static <T> ProcessorMetaSupplier supplier(
 
         return ProcessorMetaSupplier.forceTotalParallelismOne(
```

Comment:
Not related, but I just realised we have total parallelism = 1, while JDBC reading should be parallelizable without problems...

Reply:
dunno, WDYT @frant-hartm ?